tests/eas/acceptance.py

```diff
@@ -84,6 +86,52 @@ def log_result(data, log_fh):
     logging.info(result_str)
     log_fh.write(result_str)
 
+class EasTest(LisaTest):
+    """
+    Base class for EAS tests
+    """
+
+    @classmethod
+    def setUpClass(cls, *args, **kwargs):
+        conf_file = os.path.join(os.path.dirname(os.path.abspath(__file__)),
+                                 cls.conf_basename)
+
+        super(EasTest, cls)._init(conf_file, *args, **kwargs)
+
+    @memoized
+    def get_multi_assert(self, experiment, task_filter=""):
```

Comment:
I :heart: that function! And actually defining it as non private enables auto-completion in derived classes...
:+1: